src/main/java/com/bandwidth/iris/sdk/model/AvailableNpaNxx.java

```diff
@@ -1,86 +0,0 @@
-package com.bandwidth.iris.sdk.model;
-
-import com.bandwidth.iris.sdk.IrisClient;
-import com.bandwidth.iris.sdk.IrisConstants;
-import com.bandwidth.iris.sdk.IrisResponse;
-import com.bandwidth.iris.sdk.utils.XmlUtils;
-
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-/**
- * Created by sbarstow on 11/17/14.
- */
-@XmlRootElement(name = "AvailableNpaNxx")
-@XmlAccessorType(XmlAccessType.FIELD)
-public class AvailableNpaNxx {
-
-    @XmlElement(name = "City")
-    private String city;
-    @XmlElement(name = "Npa")
-    private String npa;
-    @XmlElement(name = "Nxx")
-    private String nxx;
-    @XmlElement(name = "Quantity")
-    private int quantity;
-    @XmlElement(name = "State")
-    private int state;
-
-    public static List<AvailableNpaNxx> list(IrisClient client, Map<String, Object> query)
-            throws Exception {
```

Comment:
I see what you are trying to do here. Keeping the execution code close to the model related to it. However, this makes for unfortunate code. Objects of this class are perfect "dumb" beans/domain transfer objects/models (or any other name). All they store and operate on is state.
This static method here kinda breaks that. All of the sudden the class needs to know about `IrisClient`s and various `Xml`Somethings.
Without the static method this class becomes be perfectly reusable. If a field got changed you'd only have to change this class. But with this static method the class has more than one responsibility. Not only does it store state it also interacts with the `IrisClient` and it needs to handle xml.
These xml-annotated classes are a great contribution. They are the bread and butter, but I think the interaction with the client and the xml stuff should live elsewhere.